examples/jsm/renderers/webgpu/nodes/WebGPUNodeBuilder.js

```diff
@@ -679,6 +709,35 @@ fn main( ${shaderData.varys} ) -> @location( 0 ) vec4<f32> {
 	// flow
 	${shaderData.flow}
 
+}
+`;
+
+	}
+
+	_getWGSLComputeCode( shaderData, workgroupSize ) {
+
+		return `${ this.getSignature() }
+// system
+var<private> instanceIndex : u32;
```

Comment:
I think it is better not to hardcode `instanceIndex` here, but rather use `global_invocation_id` and similar things (I think something like InvocationIDNode should be created for them).

Reply:
I like that we have a single node for use common of "call indexes", but surely we can have both.